ewfs/README.md

```diff
@@ -0,0 +1,28 @@
+# Extended Wigner's Friend Scenario (ewfs)
+
+Supplemental code for [arXiv:2409.15302](https://arxiv.org/abs/2409.15302).
+
+## Installation
+
+Python 3.12 or higher and [poetry](https://python-poetry.org/) is required. 
+
+```
+poetry install
+```
+
+## Usage
+
+In order to run the notebooks, you will need to unzip the `paper_data.zip` file.
+
+Launch the virtual environment shell via poetry:
+
+```
+poetry shell
+```
+
+Next, launch a local Jupyter notebook server.
+
+
+```
+jupyter notebook
+```
```

Comment:
Adding a one line description of the two `ipynb` files (`demo` and `plots`) could be handy.

Reply:
Good point. Done!